src/main/com/fulcrologic/fulcro/routing/dynamic_routing.cljc

```diff
@@ -127,6 +154,9 @@
       (if router-id
         (do
           (log/debug "Router" router-id "notified that pending route is ready.")
+          (when (and #?(:clj false :cljs goog.DEBUG) (nil? (get-in state-map target)))
+            (log/warn `target-ready "should route to" target "but there is no data in the DB for the ident."
```

Comment:
1. warn of error?
2. Can we think of an even better error message?

Reply:
seems ok.

Reply:
Good enough for me.